### PR TITLE
Collection of Fixes and Improvements

### PR DESCRIPTION
**Pull request by ejoerns**
* Fixes opkg pseudo wrapper creation
* Makes socket name user-specific to avoid collision
* Fixes missing env variables when running 'upgrade'
* Add support for running depmod(wrapper) in postinst scripts
* Update README

### CODE REVIEW

File: meta/classes/nfsroot.bbclass [hunk 1]
```diff
@@ -4,7 +4,7 @@ python nfsroot_eventhandler() {
         return
 
     # Set the path for the Unix socket
-    socket_path = '/tmp/nfsup.sock'
+    socket_path = f"/tmp/nfsup-{os.environ['USER']}.sock"
```
Comment (Bastian-Krause) [inline meta/classes/nfsroot.bbclass]: `os` needs to be imported before it's used.

Comment (ejoerns) [reply meta/classes/nfsroot.bbclass]: Indeed. It's there but a line too late. Fixed

File: README.nfsroot.md [hunk 2]
```diff
@@ -1,22 +1,39 @@
 # poky-nfsroot
 
 Example steps to use:
+
+Prepare your `local.conf`:
+
 ```
 echo 'INHERIT += "nfsroot"' >> $BUILDDIR/conf/local.conf
 echo 'IMAGE_FSTYPES:append = " tar.gz"' >> $BUILDDIR/conf/local.conf
-bitbake meta-ide-support
```
Comment (Bastian-Krause) [inline README.nfsroot.md:7]: For the record: no longer needed since dea0b2d2a9345e6df771aaaf2f9f2796343d6e30.

File: meta/classes/nfsroot.bbclass [hunk 3]
```diff
@@ -3,12 +3,12 @@ python nfsroot_eventhandler() {
...
         return
 
-    # Set the path for the Unix socket
-    socket_path = '/tmp/nfsup.sock'
-
     import socket
     import os
 
+    # Set the path for the Unix socket
+    socket_path = f"/tmp/nfsup-{os.environ['USER']}.sock"
```
Comment (lynxeye-dev) [inline meta/classes/nfsroot.bbclass:10]: This fails when run in a `kas` sandbox, as `USER` is not set in this environment. But I don't think we care about the username specifically here, so switching to `os.getuid()` should be fine.

Comment (ejoerns) [reply meta/classes/nfsroot.bbclass:10]: This is addressed by https://github.com/ejoerns/poky-nfsroot/pull/5 now.

**Pull request merged**